src/NServiceBus.Persistence.ServiceFabric/SagaPersister/RuntimeSagaInfo.cs

```diff
@@ -33,8 +33,8 @@ class RuntimeSagaInfo
 
             SagaAttribute = new ServiceFabricSagaAttribute
             {
-                CollectionName = !string.IsNullOrEmpty(userProvidedAttribute?.CollectionName) ? userProvidedAttribute.CollectionName : sagaDataType.Name,
-                SagaDataName = !string.IsNullOrEmpty(userProvidedAttribute?.SagaDataName) ? userProvidedAttribute.SagaDataName : sagaDataType.Name
+                CollectionName = userProvidedAttribute?.CollectionName ?? sagaDataType.Name,
```

Comment:
This change potentially introduced a bug.
When `CollectionName` or `SadaDataName` are an empty string, the override/default value won't be applied since ` userProvidedAttribute?.property` will be evaluated as non-null.